opensearch-operator/controllers/cluster_test.go

```diff
@@ -84,7 +85,37 @@ var _ = Describe("Cluster Reconciler", func() {
 		It("should apply the cluster instance successfully", func() {
 			Expect(k8sClient.Create(context.Background(), &OpensearchCluster)).Should(Succeed())
 		})
+		Sm := &monitoring.ServiceMonitor{}
```

Comment:
```suggestion
		sm := &monitoring.ServiceMonitor{}
```
Nitpick: it's a variable, so should start lowercase: